From cca63ae746db8c0cedd325d83f22e8ffa1e5fef6 Mon Sep 17 00:00:00 2001
From: Juan Gutierrez <juan.gutierrez@nxp.com>
Date: Tue, 22 Nov 2016 12:19:50 -0600
Subject: [PATCH 1312/5242] MXSCM-226 dts: imx: add pm-ignore-notify option
 for sd3 on mx6sxscm evb

commit  1c3de11b2545081a31876f52f032e0b2fb7cac5f from
https://source.codeaurora.org/external/imx/linux-imx.git

There is no card-detection pad connected for sd3 on the i.mx6 SX
EVB board. The card is assumed to be non-removable, hence, there is
no need to redetect the card during the pm callbacks. This can be
reached by including the pm-ignore-notify option on the usdhc3 device
on the dtb file.

Signed-off-by: Juan Gutierrez <juan.gutierrez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sxscm-evb.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sxscm-evb.dts b/arch/arm/boot/dts/imx6sxscm-evb.dts
index c1d0886..51c76b4 100644
--- a/arch/arm/boot/dts/imx6sxscm-evb.dts
+++ b/arch/arm/boot/dts/imx6sxscm-evb.dts
@@ -673,6 +673,7 @@
 	wp-gpios = <>;
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
+	pm-ignore-notify;
 	vmmc-supply = <>;
 	status = "okay";
 };
-- 
1.7.9.5

